docs/rpc@usage/rpcclient.py

```diff
@@ -53,7 +54,7 @@ def client_function(self, list_of_commands):
 if __name__ == "__main__":
     client =  RPCClient()
     list_of_commands = sys.argv[1:]
-    name_server = Pyro4.locateNS(host='192.168.122.34',port=9090) #Locates the name server
+    name_server = Pyro4.locateNS(host=client.dict['nameserver-address'],port=client.dict['nameserver-port']) #Locates the name server
```

Comment:
Note: This takes care of the hard coded IP address and Port address. The value is added in the dict.